tests/test_api_v1.py

```diff
@@ -524,20 +524,20 @@ def test_create_request_filter_state(app, auth_env, client, db):
             0,
         ),
         (
-            ["2021-01-01", "2021-02-02"],
+            ["2021-01-01T17:01:09.452358", "2021-02-02T17:01:09.452358"],
             {"created_from": "2021-01-01", "created_to": "2021-01-01"},
             1,
         ),
         (
             ["2021-01-01T17:01:09.452358", "2021-02-02T17:01:09.452358"],
             {
                 "created_from": "2021-01-01T17:01:09.452358",
-                "created_to": "2021-01-01T17:01:09.452358",
+                "created_to": "2021-02-02T17:01:09.452357",
             },
             1,
         ),
         (
-            ["2021-01-01", "2021-02-02"],
+            ["2021-01-01T17:01:09.452358", "2021-02-02T17:01:09.452358"],
```

Comment:
I think it would be good to keep at least one test covering `date` parameters, just to be sure they're working properly. All tests now are `datetime`. Wdyt?